### DISCUSSION TITLE
?The Youtube Key you provided was not a string.?

### DISCUSSION THREAD

**Discussion by yiaxz** [eritislami/evobot#890] [Q&A]
Hey guys im having a bit problem in here as you can see in picture above everything is correct but when i say node index.js to terminal it just says "The Youtube Key you provided was not a string." 

P.S:There is " at the end of the Youtube Token
<img width="789" alt="Ekran Resmi 2021-09-16 19 54 43" src="https://user-images.githubusercontent.com/70164464/133654396-f3efd998-a8e9-4a48-b84d-f833ec635078.png">

**Comment by eritislami** [comment]
Well if it's erroring then there is something wrong, maybe you can run it through a json validator to see if its ok
https://jsonlint.com/

**Reply by yiaxz** [reply]
![image](https://user-images.githubusercontent.com/70164464/133672307-1cb3e1fd-9605-41a9-86cb-b9df121b3ea9.png)
😶‍🌫️

**Reply by eritislami** [reply]
Honestly not sure, it should be working.

**Comment by MujiSayed** [comment]
getting the same issue on my end, but here's my workflow:

run npm install, configure config.json and add in api keys.
run npm start - bot works

run docker build -t evobot .
run docker run evobot - this results in the same error as the author.

but if i pass in my discord token and youtube api key as environment variables, it seems to work without any issues

**Reply by MujiSayed** [reply]
for anyone else that comes across this, i had to edit the .dockerignore file and remove the config.json line. after doing this, i was able to run the docker image I made without having to pass any environment variables
